packages/sources/fluent-finance/src/endpoint/balances.ts

```diff
@@ -1,10 +1,10 @@
 import { AdapterInputError, objectPath, Requester, Validator } from '@chainlink/ea-bootstrap'
 import { Config, ExecuteWithConfig, InputParameters } from '@chainlink/ea-bootstrap'
 
-export const supportedEndpoints = ['moon']
```

Comment:
Should we have a transition period where both endpoint names are supported for backwards compat? I don't know how much use this EA gets, it's easier if we don't have to but asking just in case

Reply:
I asked Michael from GTM the same thing but sounds like they're deprecating the current feed and putting up a new one for this. They also wanted to strip all references to moon because Fluent is cutting association from Moonstone Bank

Reply:
🤷‍♂️ I guess